### PR TITLE
New loop features

### PR DESCRIPTION
**Pull request by Natim**
- Domain logging: https://bugzilla.mozilla.org/show_bug.cgi?id=1246728
- Status update: https://bugzilla.mozilla.org/show_bug.cgi?id=1137849
- GA events logging: https://bugzilla.mozilla.org/show_bug.cgi?id=1243393

r? @leplatrem

### CODE REVIEW

File: source/loop/apis.rst [hunk 1]
```diff
@@ -1209,6 +1211,153 @@ Leaving the room
...
+
+Shared Domain Logs
+""""""""""""""""""
+
+    This endpoint is used to log domains that where shared with the
+    Loop-client tab sharing feature.
+
+    Request body parameters:
+
+    - **action**, Should be "logDomains" in that case.
```
Comment (leplatrem) [inline source/loop/apis.rst]: typo: `logDomain`

File: source/loop/apis.rst [hunk 2]
```diff
@@ -1209,6 +1211,153 @@ Leaving the room
...
+~~~~~~~~~~~~
+
+    **Requires authentication**
+
+    Event logging to the Google Analytics panel.
+
+    - **event**, The event name
+    - **action**, The action
+    - **label**, The label
+
```
Comment (leplatrem) [inline source/loop/apis.rst]:
```
    The endpoint will return a **204 No Content** response.

    Potential HTTP error responses include:

    - **400 Bad Request:**  Missing or invalid body parameters
```

File: source/loop/apis.rst [hunk 3]
```diff
@@ -1209,6 +1211,153 @@ Leaving the room
...
+    ::
+
+        http POST http://localhost:5000/v1/event -v \
+        event="Addon" action="start" label="Clicked Start Browsing" \
+        --auth-type=hawk --auth='ca13d91d1d4b67edf0b9523a2867b3d1b74eb63823732c441992f813f9da1f76:' --json
+
+    .. code-block:: http
+
+        POST /v1/fxa-oauth/params HTTP/1.1
```
Comment (leplatrem) [inline source/loop/apis.rst]: gni?

Comment (Natim) [inline source/loop/apis.rst]: :blush:

File: source/loop/apis.rst [hunk 4]
```diff
@@ -1209,6 +1211,153 @@ Leaving the room
...
+
+    - **event**, The event name
+    - **action**, The action
+    - **label**, The label
+
+    ::
+
+        http POST http://localhost:5000/v1/event -v \
+        event="Addon" action="start" label="Clicked Start Browsing" \
+        --auth-type=hawk --auth='ca13d91d1d4b67edf0b9523a2867b3d1b74eb63823732c441992f813f9da1f76:' --json
```
Comment (leplatrem) [inline source/loop/apis.rst]: Why don't we use `echo {} | http` as in other examples?

File: source/loop/apis.rst [hunk 5]
```diff
@@ -1209,6 +1211,153 @@ Leaving the room
...
+
+    Potential HTTP error responses include:
+
+    - **400 Bad Request:**  Missing or invalid body parameters
+
+    Example::
+
+        http POST localhost:5000/v1/rooms/pPVoaqiH89M --verbose \
+        action=status event=Session.connectionCreated state=sendrecv \
+        connections=2 sendStreams=1 recvStreams=1 \
```
Comment (leplatrem) [inline source/loop/apis.rst]: Does this generate a JSON?

Comment (Natim) [inline source/loop/apis.rst]: Yes that's one main feature of httpie: https://github.com/jkbrzt/httpie#json

File: source/loop/apis.rst (hunk 4 above)
Comment (Natim) [inline source/loop/apis.rst]: Because we do not need nested JSON for this example.

File: source/loop/apis.rst (hunk 5 above)
Comment (leplatrem) [inline source/loop/apis.rst]: cool! but you may need `--json` then!

Comment (Natim) [inline source/loop/apis.rst]: That's actually the default behavior :)

**Pull request merged**